scipy/linalg/flapack.pyf.src

```diff
@@ -735,11 +763,296 @@ interface
           :: lwork=2*minmn+MAX(maxmn,nrhs)
           ! check(lwork>=2*minmn+MAX(maxmn,nrhs))
```

Comment:
Note that I'm not saying that you should do anything extra here, I'm just pointing out directions for future work.
calc_lwork has been deprecated in current versions of scipy --- note that it copypastes parts of lapack code, which is bad, because the recommended work array sizes are not necessarily the same for different LAPACK versions.